apps/frontend/src/libs/helpers/helpers.ts

```diff
@@ -1 +1,3 @@
+export { getValidClassNames } from "./get-valid-class-name.js";
```

Comment:
```suggestion
export { getValidClassNames } from "./get-valid-class-names.js";
```
Also create a separate folder for that helper